model/commitqueue/github_pr_sender.go

```diff
@@ -66,45 +67,63 @@ func (s *githubPRLogger) Send(m message.Composer) {
 	if !s.Level().ShouldLog(m) {
 		return
 	}
-	catcher := grip.NewBasicCatcher()
+
 	msg, ok := m.Raw().(*GithubMergePR)
 	if !ok {
 		s.ErrorHandler(errors.New("message of type githubPRLogger does not contain a GithubMergePR"), m)
 		return
 	}
 
+	s.sendPatchResult(msg)
```

Comment:
There's one subscription to the version that triggers the merge sender regardless of outcome. This single sender:
1. handles sending the test results to GitHub status,
2. does the merge, and
3. reports the merge status to GitHub